Homepage/Package.resolved

```diff
@@ -0,0 +1,23 @@
+{
+  "pins" : [
+    {
+      "identity" : "mobile-ios-common",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/govuk-one-login/mobile-ios-common",
```

Comment:
Should these be here?

Reply:
I think there simply shouldn't have been a package resolved file anymore because the package has no dependencies. Seems like SPM isn't cleaning them up when dependencies are removed so I've deleted it.